notebook/static/notebook/js/actions.js

```diff
@@ -545,13 +552,13 @@ define(function(require){
         },
     };
 
-    // private stuff that prepend `.ipython` to actions names
+    // private stuff that prepend `jupyter-notebook:` to actions names
     // and uniformize/fill in missing pieces in of an action.
     var _prepare_handler = function(registry, subkey, source){
-        registry['ipython.'+subkey] = {};
-        registry['ipython.'+subkey].help = source[subkey].help||subkey.replace(/-/g,' ');
-        registry['ipython.'+subkey].help_index = source[subkey].help_index;
-        registry['ipython.'+subkey].icon = source[subkey].icon;
+        registry['jupyter-notebook:'+subkey] = {};
+        registry['jupyter-notebook:'+subkey].help = source[subkey].help||subkey.replace(/-/g,' ');
+        registry['jupyter-notebook:'+subkey].help_index = source[subkey].help_index;
+        registry['jupyter-notebook:'+subkey].icon = source[subkey].icon;
```

Comment:
Ok, it's just that the name of the category will show up in the CP, and it feel weird to actually in UI, and it feel weird to precise that this apply to "jupyter notebook"
![screen shot 2015-10-12 at 18 01 55](https://cloud.githubusercontent.com/assets/335567/10442839/5e28140c-710b-11e5-949c-6369eb921b3f.png). 

Feel a bit like [Truman show](https://youtu.be/Inscky6EyQ8?t=1m31s).